Formula/imagemagick.rb

```diff
@@ -72,49 +73,51 @@ def install
       --enable-static
     ]
 
-    if build.without? "modules"
-      args << "--without-modules"
-    else
-      args << "--with-modules"
-    end
-
-    if build.with? "opencl"
-      args << "--enable-opencl"
-    else
-      args << "--disable-opencl"
-    end
-
-    if build.with? "openmp"
-      args << "--enable-openmp"
-    else
-      args << "--disable-openmp"
-    end
-
-    if build.with? "webp"
-      args << "--with-webp=yes"
-    else
-      args << "--without-webp"
+    %W[
+      hdri hdri
+      opencl opencl
+      openmp openmp
+      zero-configuration zero-configuration
+    ].each_slice(2) do |option, feature|
+      if build.with? option
+        args << "--enable-#{feature}"
+      else
+        args << "--disable-#{feature}"
+      end
     end
 
-    if build.with? "openjpeg"
-      args << "--with-openjp2"
-    else
-      args << "--without-openjp2"
+    %W[
+      fftw fftw
```

Comment:
let's factor out the ones where the pairs are the same https://gist.github.com/ilovezfs/8300cbda17d6bedeacbaab5d90494d74

also I prefer %w if there's no interpolation